src/content/docs/en/reference/cli-reference.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: CLI Reference
+title: CLI Commands
```

Comment:
The back half of this doc does feel "referencey." The `astro dev` command was already introduced in `Installation`, so I don't see the need to move the CLI reference into Getting Started. I'd rather explain the small amount they need to know about the CLI in `Installation` - the content you added is great, but it makes more sense there.

That said, it's NWTWWHB!